package.json

```diff
@@ -28,28 +28,29 @@
     "prebuild": "rm -rf dist",
     "lint": "eslint .",
     "prettier": "prettier --write \"**/*.{js,json,css,scss,less,md,ts,html}\"",
-    "update": "npx npm-check -u"
+    "update": "npx npm-check -u",
+    "prepare": "husky install"
   },
   "dependencies": {
-    "angular": "npm:@xlts.dev/angular@1.8.7",
-    "angular-animate": "npm:@xlts.dev/angular-animate@1.8.7",
-    "angular-aria": "npm:@xlts.dev/angular-aria@1.8.7",
-    "angular-material": "^1.2.3",
-    "angular-messages": "npm:@xlts.dev/angular-messages@1.8.7"
+    "angular": "npm:@xlts.dev/angular@^1.9.1",
+    "angular-animate": "npm:@xlts.dev/angular-animate@^1.9.1",
+    "angular-aria": "npm:@xlts.dev/angular-aria@^1.9.1",
+    "angular-material": "npm:@xlts.dev/angular-material@^1.2.4",
```

Comment:
Actually, I think we want to use AngularJS Material here (instead of XLTS for AngularJS Material) to demonstrate compatibility.

In any case, if we decide to switch to XLTS for AngularJS Material, we should update docs (for example `README.md`) to mention that instead of AngularJS Material.

Reply:
No problem, I can change it. I just don't know if that was the intention of this project. @Splaktar, any thoughts?

Reply:
I think that this was created before XLTS for AngularJS existed. Please more forward with updating to the XLTS.dev version of AngularJS Material. Thanks!

Reply:
@jzolnowski, keep in mind that we should update the README (and any other references - if any) to mention XLTS for AngularJS Material and not the LTS AngularJS Material.

Maybe link to https://material.angularjs.xlts.dev/latest.